docs/models/overrides.md

```diff
@@ -0,0 +1,52 @@
+Item Property Overrides
```

Comment:
I want to keep the separate, because the adv model section is meant for those who want to actually go in and do fancy things with the system, while the base models section is just for those who want to slap some JSON together and just "have it work."